cypress/e2e/help-section/helpAreas.feature

```diff
@@ -26,10 +31,24 @@ Feature: Help Section - To use the help areas in dBildungscloud
         When I click on button Submit to send form
         Then I see message '<feedback_message>'
 
+        # submit a request via contact form inside help area
+        Given I am logged in as a '<student>' at '<namespace>'
+        When I click on Help Section in sidebar
```

Comment:
I think we need to remove this step because in the later step we are using contact form instead of help section?

Reply:
I had to remove it because now teacher is also sending the request form and so the page can stay open and teacher can go on there.